libs/state/README.md

```diff
@@ -278,16 +63,16 @@ export class StatefulComponent {
 }
 ```
 
-In very small components you can also extend to keep the api short.
+### Extend (minimal)
 
-**Extend from the `RxState` inside a `Component`, `Directive` or `Service`**
+If you wish, there is also the possibility of **extending** the `RxState` service. This is generally
```

Comment:
I think both ways are recommended but for different use cases. We could rephrase it to be "equally recommended" for both situations.

Reply:
yes, it's just another way with different pros and cons.